tests/test_pipenv.py

```diff
@@ -472,8 +506,9 @@ def test_editable_vcs_install(self, pip_src_dir, pypi):
 
     @pytest.mark.install
     @pytest.mark.pin
-    def test_windows_pinned_pipfile(self, pypi):
-        with PipenvInstance(pypi=pypi) as p:
+    @pytest.mark.skipif(not WE_HAVE_INTERNET, reason='requires internet to resolve')
```

Comment:
This one is failing. The mock PyPI only has table 0.12.1 and cannot resolve `<0.12`. I don’t know why Travis did not complain.

Reply:
@uranusjr I added one lower than 0.12.1

Reply:
uh, and totally committed and pushed that....